css/mistica-common.css

```diff
@@ -0,0 +1,158 @@
+/* Default text color */
+[data-mistica-skin] {
+    color: var(--mistica-color-textPrimary);
+    background: var(--mistica-color-background);
+}
+
+/* border-radius utility classes */
+.mistica-border-radius-avatar {
```

Comment:
I'm not sure if border-radius utility classes are worth, maybe having the css vars is enough

Reply:
What is the use case? It seems like the CSS var could be enough

Reply:
agree. If we need the class in the future, we can add it, but for now I think we can live with the vars